### ISSUE TITLE
Non-AEP customer updates

### ISSUE THREAD

**Issue by dhwoolsey** [AdobeDocs/platform-learn.en#33]
Issue in ./help/tutorial-web-sdk/configure-identities.md

Step 2 (picking a sandbox) doesn't apply to customers without an AEP license

It may also be important to call out that namespace identities can't be deleted. So for customers without an AEP license, this step will permanently create an identity namespace, even when they upgrade to AEM. Also, if a number of users at the same company do this tutorial, they will all see this this namespace and may not need to create it. 

This section may also be a good place to start calling out that any AAM cross-device data sources will show up in the identity namespaces. And if they are using AAM with AEP, it may be better to simply use an existing AAM-created namespace as opposed to creating a net new one just for this tutorial. 

What's also true is that it may be easier if the customer has an AAM license to create a cross-device data source in AAM just for this tutorial. It'll show up in the identity namespace and then when they delete the namespace on the AAM side after they've finished the tutorial, it will actually delete it from AEP identities as well. That way, the only customers that may need to create a luma identity namespace and have it permanently in their production sandbox will be non-AAM customers going through this tutorial.

**Comment by dwright-adobe** [comment]
Updated the sandbox note to:
"If you are the customer of a Platform-based application like Real-time CDP, we recommend using a development sandbox for this tutorial. If you are not, use the Prod sandbox."

The sandbox name is still visible, so this will at least let these customers know what that signifies.

Good point on deleting namespaces. I will add that.

With the AAM namespaces, is that dependent on their being a Platform app customer with the Audience Manager connector enabled? What needs to be provisioned or configured for that to happen?

**Comment by dhwoolsey** [comment]
No, it's not. In April of 2019, all existing AAM Cross-Device data sources had a corollary UIS namespace created. Since then, any new AAM Cross-Device data sources also created UIS namespaces, regardless of AEP license. We are essentially holding them in UIS so that when customers move to AEP, they are already there and ready to go. This is why existing AAM data sources will be visible.
